### PR TITLE
CKS task19 solution has been expanded

### PR DESCRIPTION
**Pull request by devopskupryk**

### CODE REVIEW

**Comment by ViktorUJ** [comment]
@devopskupryk   , please leave the previous options and add your own as alternatives.

**Comment by devopskupryk** [comment]
@ViktorUJ 
I've returned these commands:
```
RUN groupadd myuser
RUN useradd -g myuser myuser
```
Or do you mean some other options?

File: tasks/cks/labs/19/SOLUTION.MD [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
 podman build . -t cks:14
+```
 
+# Run a container
+```
+podman run -d --rm --name cks-14 cks:14
+```
 
-podman  run -d --name  cks-14   cks:14
-sleep 2
```
Comment (ViktorUJ) [inline tasks/cks/labs/19/SOLUTION.MD:20]: Sometimes this won't work, as the container may not have time to start. that's why sleep was added.

File: tasks/cks/labs/19/SOLUTION.MD [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
+# Fix dockerfile
+```
+vim  /var/work/14/Dockerfile
+```
+*It should looks like this one:*
+```
 FROM ubuntu:20.04
-RUN apt-get update
-RUN apt-get -y install curl
+RUN apt-get update && apt-get -y install curl
```
Comment (ViktorUJ) [inline tasks/cks/labs/19/SOLUTION.MD:10]: What's the sense of this fix if it's the same?

File: tasks/cks/labs/19/SOLUTION.MD (hunk 1 above)
Comment (devopskupryk) [reply tasks/cks/labs/19/SOLUTION.MD:20]: Sleep has been returned

File: tasks/cks/labs/19/SOLUTION.MD (hunk 2 above)
Comment (devopskupryk) [reply tasks/cks/labs/19/SOLUTION.MD:10]: This isn't the same.
Check [best practices](https://docs.docker.com/build/building/best-practices/#apt-get) please

Comment (ViktorUJ) [reply tasks/cks/labs/19/SOLUTION.MD:10]: There is such a question in the real exam . and there you only need to add creating a user . and running the app as that user . changing other lines of the dockerfile may cause this question to fail .

**Pull request closed without merging**